client/utils/tags.tsx

```diff
@@ -95,7 +95,7 @@ export function injectConditionalTags(
   }
 
   // inject cap and Chinese gap for production Chinese
-  if (parsedHomeUrl.subdomain === 'chinese' && parsedHomeUrl.tld === 'org') {
+  if (homeLocation.split('/')[1] === 'chinese' && parsedHomeUrl.tld === 'org') {
```

Comment:
The issue seems to be that `homeLocation` can be `'https://www.freecodecamp.org'`, `'https://freecodecamp.org/chinese'`, or any of the other URL strings in the unit tests in `tags.test.tsx`.

But when splitting one of those URL strings on `'/'`, the element at index 1 is an empty string.

This is not the DRYest solution, but we could just use the `pop` method to get the last element of the split string array:

```suggestion
  if (homeLocation.split('/').pop() === 'chinese' && parsedHomeUrl.tld === 'org') {
```